packages/lazy-images/.gitattributes

```
@@ -1,4 +1,5 @@
 # Files not needed to be distributed in the package.
+.eslintrc.js        export-ignore
 .gitattributes      export-ignore
```

The don't need .gitignore either.
```suggestion
.eslintrc.js        export-ignore
.gitattributes      export-ignore
.gitignore          export-ignore
```